src/lib/styles/foundation/partyColors.scss

```diff
@@ -19,6 +19,26 @@
   --noc: #c8c8c8;
 
   --undeclared: #e7e7e7;
+
```

Comment:
These are a secondary color for each UK party - in light and dark mode. Colors come from here: https://www.figma.com/design/RZHnmwc2tmB6j1k4bNtyND/Style-Guide?node-id=2527-2&t=Zy134jAdsgL06kao-0